docker/Dockerfile.nvidia

```diff
@@ -0,0 +1,25 @@
+FROM nvidia/cuda:11.8.0-cudnn8-runtime-ubuntu22.04
```

Comment:
Apply all the changes from the main Dockerfile

Reply:
I don't understand this comment. Which changes? The Dockerfiles for cuda and cpu are different, because there is no common base. cuda image is based on nvidia/cuda and the cpu image is based on python.